gwcs/conftest.py

```diff
@@ -27,7 +27,3 @@
     TESTED_VERSIONS[packagename] = version.version
 except NameError:   # Needed to support Astropy <= 1.0.0
     pass
-
-pytest_plugins = [
-    'asdf.tests.schema_tester'
-]
```

Comment:
Instead of being removed, the schema tester plugin should be moved to a top-level `conftest.py`.  Reference how it is done in `jwst` so that it is future-proof once a new release of `asdf` happens, which now installs the plugin via `entry_points`.

https://github.com/spacetelescope/jwst/blob/f1bbfdd4a8bc2979eeff8ee27c18e4ebd689410d/conftest.py#L1-L8

The `astropy` conftest.py is similar.